tools/topology/topology2/cavs/CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+# Array of "input-file-name;output-file-name;comma separated pre-processor variables"
+set(TPLGS
+# CAVS HDMI only topology with passthrough pipelines
+"cavs-passthrough-hdmi\;cavs-passthrough-hdmi"
+# CAVS HDA topology with passthrough pipelines
+"cavs-passthrough-hdmi\;cavs-passthrough-hda\;HDA_CONFIG=passthrough"
+# CAVS HDMI only topology with gain-based pipelines
+"cavs-gain-hdmi\;cavs-gain-hdmi"
+# CAVS HDA topology with gain-based pipelines
+"cavs-gain-hdmi\;cavs-gain-hda\;HDA_CONFIG=gain"
+# CAVS HDA topology with mixer-based pipelines
+"cavs-passthrough-hdmi\;cavs-mixin-mixout-hda\;HDA_CONFIG=mix"
+)
+
+# generate ABI
+execute_process(
+	COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/get_abi.sh ${SOF_ROOT_SOURCE_DIRECTORY}
+	OUTPUT_FILE ${CMAKE_CURRENT_BINARY_DIR}/abi.conf
+)
+
+add_custom_target(topology2_cavs ALL
+    COMMAND ${CMAKE_COMMAND} -E copy_directory . ${SOF_TOPOLOGY_BINARY_DIRECTORY}
```

Comment:
Is this the "silent" v1 -> v2 upgrade? If yes add a short comment.

Assuming there will be other subdirectories later besides `cavs/`, can this be done only once for all in the `../CMakeLists.txt`?